tennis-scheduler-web-app/src/components/Navbar.jsx

```diff
@@ -64,6 +65,7 @@ export const NavbarStart = () => {
                 <>
                   <Nav className="me-auto">
                     <Link className="nav-link" to="/profile">Profile</Link>
+                    <Link className='nav-link' to="/players">Players</Link>
```

Comment:
Why here Players instead of Tennis players like in admin section?

Reply:
By accident